tests/testthat/test-bug_search.R

```diff
@@ -8,16 +8,14 @@ valid_parameters <- c("alias", "assigned_to", "component", "creation_time",
 
 # test_that("bug_search works", {
 #     vcr::use_cassette("bug_search", {
-#         bs <- bug_search()
+#         bs <- bug_search(id=1)
 #         url <- "https://bugs.r-project.org/bugzilla/rest/bug?id=1"
-#         params <- params(1)
 #     })
-#     expect_equal(missing_host(), "https://bugs.r-project.org/bugzilla/")
+#     expect_equal(paste0(missing_host(), "rest/bug?", params(id=1)), "https://bugs.r-project.org/bugzilla/rest/bug?id=1")
 # })
 
 test_that("params works", {
-    vcr::use_cassette("params", {
-        par <- params()
-    })
+    par <- params()
     expect_true(all(names(par) %in% valid_parameters))
+    expect_true(all(!names(params("ids")) %in% valid_parameters))
```

Comment:
I don't understand here what is being tested. `params` is a helper function to collapse parameters it should never have names. But maybe it needs to be improved as `params("ids")` should have raised an error. 

The idea of params is to be used like this: `params(id = c(15, 20))` and return the parameters ready for adding them to url. Anyway, this is not critical and can be fixed on a later PR.

Reply:
Here, I was checking if the condition gets true like "ids" is not present in the valid_parameters which can help to test the stop condition. As you mentioned I look into the parameters.

Reply:
If you want to test if it raises an error you should use something like `expect_error(params(ids=1))`